README.md

```diff
@@ -0,0 +1,28 @@
+# eslint-config-node
+
+Keboola Node.js coding standard
+
+## Usage
+
+1. Install: `yarn add --dev eslint eslint-plugin-import @keboola/eslint-config-node`
+2. Add `keboola-node` to the plugins section of your `.eslintrc.json` configuration file.:
+
+```json
+{
+    "plugins": [
+        "keboola-node"
+    ]
+}
+```
+
+## Standard
+
+The standard extends [AirBnB coding standard](https://github.com/airbnb/javascript) with these exceptions:
+
+- `strict` is disabled. You are not required to define strict mode explicitly as it is switched on by default inside modules and classes and we expect that you write a modern code with OOP.
```

Comment:
Som si prave myslel, ze OOP je uz zastarale :)

Reply:
Na organizaci a přehlednost kódu imho ne, v rámci metod ať už si člověk programuje klidně funkcionálně, ale to je asi na jinou diskuzi. Tu druhou půlku věty jsem teda oddělal. ;o)

Reply:
Suhlasim s tym co spises a takto je to lepsie.